recipes/jurigged/recipe.yaml

```diff
@@ -0,0 +1,59 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: jurigged
+  version: "0.6.1"
+  python_min: '3.10'
```

Comment:
This only needs to be set if it is greater than the current conda-forge default for `python_min`, so this line can be removed